gems/laa_multi_step_forms/app/presenters/tasks/base_task.rb

```diff
@@ -9,8 +9,9 @@ def initialize(application:)
     end
 
     def self.build(name, **)
-      class_name = "Tasks::#{name.to_s.split('.').first.camelize}"
-
+      parts = name.to_s.split('/')
+      namespace = parts.first.camelize if parts.count == 2
+      class_name = "#{namespace}::Tasks::#{parts.last.split('.').first.camelize}"
```

Comment:
🎖️

Reply:
have given prior-authority, as it currently stands, a once over on dev crm4 env and no problems encountered 👍